tools/docker/Dockerfile.centos6

```diff
@@ -23,15 +23,19 @@ LABEL Vendor="Apache.org" License="ApacheV2" Version="4.6.0"
 ENV PKG_URL=http://jenkins.buildacloud.org/job/package-rhel63-master/lastSuccessfulBuild/artifact/dist/rpmbuild/RPMS/x86_64
 
 # install CloudStack
-RUN yum install -y \
+RUN yum install -y nc wget \
     ${PKG_URL}/cloudstack-common-4.6.0-SNAPSHOT.el6.x86_64.rpm \
     ${PKG_URL}/cloudstack-management-4.6.0-SNAPSHOT.el6.x86_64.rpm
 
 RUN cd /etc/cloudstack/management; \
     ln -s tomcat6-nonssl.conf tomcat6.conf; \
     ln -s server-nonssl.xml server.xml; \
-    ln -s log4j-cloud.xml log4j.xml
+    ln -s log4j-cloud.xml log4j.xml; \
+    cd /usr/share/cloudstack-common/scripts/vm/hypervisor/xenserver; \
+    wget http://download.cloud.com.s3.amazonaws.com/tools/vhd-util
```

Comment:
vhd-utils is added because it's really not fun adding it once deployed. it also does not break support for KVM.

As this dockerfile is not using noredis build, should we have a seperate image including all the noredis which would also include vhd-utils?   if so, the basic build would only work with KVM then :-S